### DISCUSSION TITLE
Problemas con internet y WSL

### DISCUSSION THREAD

**Discussion by Burbina200** [IIC2133-PUC/2024-1#15] [Ayuda]
Holaa! Estuve teniendo problemas para instalar WSL en mi PC y en mi notebook. Me tiraba errores de que no era posible comunicarse con los servidores de Ubuntu. Para solucionarlo tuve que compartir internet desde mi celular (Al parecer el error era debido a la red de mi casa)

Actualmente tengo todo bien configurado en el PC y el notebook, pero ahora tengo otro problema: Cuando intento hacer push para subir mis avances de la T0 al repositorio tampoco me deja (Esto solo con el internet de mi casa, con otros si puedo) ¿Por que motivo el internet de mi casa no me deja utilizar WSL? Espero puedan ayudarme.

**Comment by CataAlamos** [comment]
Hola! Hiciste alguna modificación al DNS?

**Comment by Burbina200** [comment]
Hola, buenas. No, ninguna.
________________________________
De: Cata Alamos ***@***.***>
Enviado: domingo, 17 de marzo de 2024 18:20
Para: IIC2133-PUC/2024-1 ***@***.***>
Cc: Benjamín Alejandro Urbina Fuenzalida ***@***.***>; Author ***@***.***>
Asunto: Re: [IIC2133-PUC/2024-1] Problemas con internet y WSL (Discussion #15)


Hola! Hiciste alguna modificación al DNS?

**Reply by CataAlamos** [reply]
Hola! Intentaste esto?
https://www.enmimaquinafunciona.com/pregunta/228986/no-hay-conexion-a-internet-ubuntu-wsl-mientras-vpn

También, puedes subir el mensaje de error que te da?

**Comment by Burbina200** [comment]
Lo intentaré en la tarde al llegar a mi casa.

Gracias!!
________________________________
De: Cata Alamos ***@***.***>
Enviado: lunes, 18 de marzo de 2024 7:52
Para: IIC2133-PUC/2024-1 ***@***.***>
Cc: Benjamín Alejandro Urbina Fuenzalida ***@***.***>; Author ***@***.***>
Asunto: Re: [IIC2133-PUC/2024-1] Problemas con internet y WSL (Discussion #15)


Hola! Intentaste esto?
https://www.enmimaquinafunciona.com/pregunta/228986/no-hay-conexion-a-internet-ubuntu-wsl-mientras-vpn
